content/en/user-guide/integrations/hyperexecute/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: HyperExecute
```

Comment:
nit: Should we include "LambdaTest" in the title? HyperExecute by itself is probably not well-known by most folks in our community..